test/benchmarks/run.sh

```diff
@@ -39,22 +39,28 @@ trap cleanup SIGINT SIGTERM
 
 solc="${SOLIDITY_BUILD_DIR}/solc/solc"
 benchmarks_dir="${REPO_ROOT}/test/benchmarks"
-chains_sol="${benchmarks_dir}/chains.sol"
 time_bin_path=$(type -P time)
 
-solc_command_legacy=("${solc}" --optimize --bin "${chains_sol}")
-solc_command_via_ir=("${solc}" --via-ir --optimize --bin "${chains_sol}")
+for input_file in "chains.sol" "OptimizorClub.sol"
```

Comment:
It's fine this way if we don't add more benchmarks, but if we're planning to do so, I'd prefer to have this declared in an array, and then to iterate over the contents of said array, i.e.
```
for input_file in "${benchmarks[@]}"
do
   ....
done
```